### PR TITLE
docs: change udf and faq level, add sql guide

### PR DESCRIPTION
**Pull request by vagetablechicken**
udf & faq level
sql guide, link openmldb sql emulator
some desc in must read

### CODE REVIEW

File: hybridse/tools/documentation/udf_doxygen/Makefile
Comment (aceforeverd) [inline hybridse/tools/documentation/udf_doxygen/Makefile:1]: fix the `baseUrl` in `hybridse/tools/documentation/udf_doxygen/config.json`, should be the prefix path from the web url

File: docs/zh/deploy/install_deploy.md
Comment (lumianph) [inline docs/zh/deploy/install_deploy.md]: what is this? uncompleted new section?

**Review by lumianph — CHANGES_REQUESTED** [review]

File: docs/zh/developer/built_in_function_develop_guide.md
Comment (lumianph) [inline docs/zh/developer/built_in_function_develop_guide.md]: change to a relative path?

File: docs/zh/quickstart/beginner_must_read.md
Comment (lumianph) [inline docs/zh/quickstart/beginner_must_read.md]: the full name can be "OpenMLDB SQL Emulator"
emulator should be the correct word

File: hybridse/tools/documentation/udf_doxygen/Makefile
Comment (vagetablechicken) [reply hybridse/tools/documentation/udf_doxygen/Makefile:1]: But it will be `/openmldb_sql/Files/udfs_8h.md#function-abs`? We want no `Files/`

File: docs/zh/deploy/install_deploy.md
Comment (vagetablechicken) [reply docs/zh/deploy/install_deploy.md]: I want to add some links for cluster maintainers, lost the patch. It's better to add after full inspect merged. Revert now.

File: docs/zh/quickstart/beginner_must_read.md
Comment (vagetablechicken) [reply docs/zh/quickstart/beginner_must_read.md]: done

File: hybridse/tools/documentation/udf_doxygen/Makefile
Comment (aceforeverd) [reply hybridse/tools/documentation/udf_doxygen/Makefile:1]: doxybook2 seems always output to `Files` subdirectory

Comment (vagetablechicken) [reply hybridse/tools/documentation/udf_doxygen/Makefile:1]: mark it unsolved

**Review by aceforeverd — APPROVED** [review]

**Review by lumianph — APPROVED** [review]

**Pull request merged**
